lib/catData.dart

```diff
@@ -0,0 +1,49 @@
+class CatData {
```

Comment:
Try to add some structure to the project.
move CatData to lib/models/ package.

Reply:
fixed